danjones5071/shrapnel
# SHRPX-3 Generate Rails project

:pushpin: closes #3.

**What was changed and why it was necessary:**
The basic structure for the Rails backend and React frontend is introduced to the project. This is a necessary starting point to begin developing new features for the application.

**How to test the changes:**
1. Run `rails s` to launch the application.
2. Navigate to http://localhost:3000 and confirm that the phrase "Hello React!" displays in the browser.
